agents-api/agents_api/routers/sessions/session.py

```diff
@@ -52,6 +55,30 @@
 )
 
 
+def cache(f):
+    async def wrapper(
+        self, init_context: list[ChatML], settings: Settings
+    ) -> ChatCompletion:
+        key = xxhash.xxh64(
+            json.dumps(
```

Comment:
This causes `TypeError: Object of type UUID is not JSON serializable`.

Particularly due to the `id` in the context which is UUID.

Should be string when converting to json